modules/shell/command.go

```diff
@@ -176,7 +176,7 @@ func readData(t testing.TestingT, log *logger.Logger, reader *bufio.Reader, writ
 			break
 		}
 
-		log.Logf(t, line)
+		log.Logf(t, "%s", line)
```

Comment:
Just to make sure I understand:

1. In #982, you show an example where the log output contains what looks like broken string interpolation. I can't see any reason you'd expect interpolation to work, so I'm guessing the real goal is to _disable_ interpolation, so that random `%xxx` style codes in stdout/stderr don't accidentally get interpreted as interpolation, right?
2. If so, wouldn't switching to `log.Log` instead of `log.Logf` be the proper fix here?

Reply:
> 2\. If so, wouldn't switching to `log.Log` instead of `log.Logf` be the proper fix here?

Yeh, I also assumed that at first, but `logging.Logger` only has `Logf`.

Reply:
Ah, OK, got it. In that case, could you add a comment above this line explaining:

1. Why we need this "%s" indirection (there's only Logf, no Log)
1. Pointing back to this PR so maintainers know what issue this is fixing.

Thanks!

Reply:
Done.

Reply:
Perfect, thank you!